notebooks/0103_GFED4-BB4CMIP-prepare.py

```diff
@@ -67,9 +67,9 @@
         "unit_label": "Mt BC / yr",
         "filename_label": "BC",
     },
-    "NMVOC": {
+    "NMVOCbulk": {
```

Comment:
I would have just kept "NMVOC", but maybe you're seeing something I'm not?

Reply:
I think this doesn't work with `bb4cmip_files` at the moment

Reply:
Can probably be changed if we want to make this clearer some other time, but I assume this is fine for now (it's being renamed in 0104).